rotkehlchen/tests/utils/history.py

```diff
@@ -736,6 +736,81 @@ def assert_kraken_asset_movements(
     assert_asset_movements(expected, to_check_list, deserialized, movements_to_check)
 
 
+def assert_ignored_btc_asset_movements(
```

Comment:
do not do it like this. If it's not used in at least 2 places do not create a new function.